src/jit/liveness.cpp

```diff
@@ -2016,13 +2057,24 @@ void Compiler::fgComputeLifeLIR(VARSET_TP& life, BasicBlock* block, VARSET_VALAR
                         // these nodes.
```

Comment:
@BruceForstall do you know should `PUTARG_SPLIT` be here as well? I suppose it changes stack level and we should check it too.